src/pages/[platform]/build-a-backend/storage/list-files/index.mdx

```diff
@@ -109,6 +118,61 @@ function processStorageList(response) {
 
 This places each item's data inside a special `__data` key.
 
+### Excluding subpaths
+
+In addition to using the `list` API to get all the contents of a path, you can also use it to get only the files within a path while excluding files under subpaths.
+
+For example, given the following keys in your `path` you may want to return only the jpg object, and not the "vacation" subpath and its contents:
+
+```
+photos/photo1.jpg
+photos/vacation/
+```
+
+This can be accomplished with the `subpathStrategy` option:
+
+```js
+const result = await list({ 
+  path: "photos/",
+  options:{
+    subpathStrategy: { strategy:'exclude' }
+  }
+});
+```
+
+The response will include only the objects within the `photos/` path and will also communicate any excluded subpaths:
+
+```js
+{
```

Comment:
is this intended to show a json blob?

Reply:
Yes, following the pattern already used on line 54.